tests/flytekit/integration/remote/workflows/basic/container_workflow.py

```diff
@@ -0,0 +1,85 @@
+import os
+import docker
+import logging
+from pathlib import Path
+
+import flytekit
+from flytekit import ContainerTask, kwtypes, workflow, task
+from flytekit.types.file import FlyteFile
+from flytekit.types.directory import FlyteDirectory
+
+
+logger = logging.getLogger(__file__)
+
+client = docker.from_env()
+path_to_dockerfile = "tests/flytekit/unit/core/"
+dockerfile_name = "Dockerfile.raw_container"
+client.images.build(path=path_to_dockerfile, dockerfile=dockerfile_name, tag="flytekit:rawcontainer")
+
+flyte_file_io = ContainerTask(
```

Comment:
I think the only way to do this is to build an image at CI.
We can use my image `futureoutlier/rawcontainer:0320` first.

Reply:
integration tests assume that the demo cluster [is running](https://github.com/flyteorg/flytekit/blob/master/.github/workflows/pythonbuild.yml#L276-L278), so it's safe to build an image and push to its registry (i.e. `localhost:30000`).